src/exe_reader.cpp

```diff
@@ -28,7 +28,7 @@
 
 namespace {
 	// hashes of known RPG_RT startup logos
-	std::array<uint32_t, 33> logo_crc32 = {
+	std::array<uint32_t, 36> logo_crc32 = {
```

Comment:
With c++17 we can use type deduction and  remove the explicit array size.

`std::array logo_crc32{ val1, val2, ... };`

Reply:
good point. I had to add ``u`` suffix to all the hex numbers because some of them are unsigned but this is much better than counting :).